pulsar-broker-common/src/main/java/org/apache/pulsar/broker/web/plugin/servlet/AdditionalServlets.java

```diff
@@ -23,6 +23,7 @@
 import java.util.Map;
 import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang.StringUtils;
```

Comment:
Please use `org.apache.commons.lang3.StringUtils` instead of `org.apache.commons.lang.StringUtils`